dev/Repeater/TestUI/Samples/LayoutSamples/NonVirtualLayoutPages/NonVirtualStackLayoutSamplePage.xaml

```diff
@@ -5,13 +5,232 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:controls="using:Microsoft.UI.Xaml.Controls"
     xmlns:local="using:MUXControlsTestApp.Samples">
-    <Grid>
-        <ScrollViewer>
-            <controls:ItemsRepeater x:Name="repeater">
-                <controls:ItemsRepeater.Layout>
-                    <local:NonVirtualStackLayout />
-                </controls:ItemsRepeater.Layout>
-            </controls:ItemsRepeater>
-        </ScrollViewer>
-    </Grid>
+   <!-- <StackPanel>
+        <StackPanel.Resources>
```

Comment:
you should create a separate page for flex layout since this file is for NonVirtualStackLayout